tests/fixtures/expected/toctree/index.html

```diff
@@ -17,12 +17,18 @@ <h1>Toctree<a class="headerlink" href="#toctree" title="Permalink to this headli
 </div>
 <div class="toctree-wrapper compound">
     <ul>
-        <li><a href="file.html#title">Title</a></li>
+        <li><a href="file.html">Title</a></li>
+    </ul>
+</div>
+<div class="toctree-wrapper compound">
+    <ul>
+        <li><a href="directory/another_file.html">Another file</a></li>
     </ul>
 </div>
 <div class="toctree-wrapper compound">
     <ul>
-        <li><a href="directory/another_file.html#another-file">Another file</a></li>
+        <li><a href="file.html">Title</a></li>
```

Comment:
either we need to add a `:maxdepth: 1` option in the rst file, or trhe expectation file is missing 
```html
            <ul>
                <li><a href="file.html#sub-title">Sub title</a></li>
            </ul>
```

Reply:
I don't understand your comment 😐  The original RST toctree is:

```rst
.. toctree::

    directory/another_file
```

And that file doesn't have subtitles, right?

Reply:
actually this line referes to the toctree you've added:
```rst
.. toctree::

    file
    :ref:`Some specific reference <some_reference>`
```
and `file.rst` had a subtitle.
But I think we can remove this new toctree, since we decided not to handle the `:ref:` nested into a toctree